src/Home.js

```diff
@@ -111,20 +159,54 @@ const styles = StyleSheet.create({
     color: "white"
   },
   buttonText: {
-    fontSize: 18,
+    fontSize: 16,
     fontWeight: "600",
     padding: 15,
     alignSelf: "center",
     color: "#fff"
   },
-  button: {
-    position: "absolute",
-    bottom: 30,
+  buttonContainer: {
     backgroundColor: "#4696ec",
     width: 150,
     alignSelf: "center",
     borderRadius: 25,
+    shadowOpacity: 0.3,
+    shadowRadius: 2,
+    marginTop: 20,
+    shadowOffset: {
+      height: 1,
+      width: 1
+    }
+  },
+  floatingButton: {
+    position: "absolute",
+    bottom: 30,
+  },
+  inputView: {
+    height: 40,
+    margin: 15,
+    borderWidth: 1,
+    padding: 10,
+  },
+  modalView: {
+    backgroundColor: "white",
+    borderRadius: 20,
+    padding: 40,
+    alignSelf: "center",
+    justifyContent: "center",
+    shadowColor: "#000",
+    width: 325,
     shadowOpacity: 0.5,
     shadowRadius: 5,
+    marginTop: 300
+  },
+  modalDismiss: {
+    position: "absolute",
+    right: 20,
+    top: 15
   },
+  modalDismissText: {
+    fontSize: 20,
```

Comment:
A bit concerned abt how many different values of `fontSize` we have in this file, we should probably audit in addition to the style naming